test/react-native-cli/features/steps/steps.rb

```diff
@@ -100,14 +106,15 @@
 end
 
 def parse_package_json
-  length_before = Runner.interactive_session.stdout_lines.length
+  stdout_lines = Runner.interactive_session.stdout_lines
+  length_before = stdout_lines.length
 
   steps %Q{
     When I input "cat package.json" interactively
     Then I wait for the shell to output '"dependencies": \{' to stdout
   }
 
-  after = Runner.interactive_session.stdout_lines[length_before..]
+  after = stdout_lines[length_before..stdout_lines.length]
```

Comment:
The endless range should do the same thing as this; was it broken?

Reply:
Endless ranges were added in Ruby 2.6 and the Debian 10 image only has 2.5 on it.  I could probably have worked harder and found a newer image, but it felt like diminishing returns when I could just tweak the code.

Reply:
Yeah that makes sense, just checking I understood why